code/modules/clothing/neck/ties.dm

```diff
@@ -38,34 +38,28 @@
 	if(ishuman(M) && isliving(user))
 		if(user.a_intent == INTENT_HELP)
 			var/body_part = parse_zone(user.zone_selected)
-			if(body_part)
-				var/their = "their"
-				switch(M.gender)
-					if(MALE)
-						their = "his"
-					if(FEMALE)
-						their = "her"
-
-				var/sound = "pulse"
-				var/sound_strength
-
-				if(M.stat == DEAD || (M.status_flags&FAKEDEATH))
-					sound_strength = "cannot hear"
-					sound = "anything"
-				else
-					sound_strength = "hear a weak"
-					switch(body_part)
-						if("chest")
-							if(M.oxyloss < 50)
-								sound_strength = "hear a healthy"
-							sound = "pulse and respiration"
-						if("eyes","mouth")
-							sound_strength = "cannot hear"
-							sound = "anything"
-						else
-							sound_strength = "hear a weak"
-
-				user.visible_message("[user] places [src] against [M]'s [body_part] and listens attentively.", "You place [src] against [their] [body_part]. You [sound_strength] [sound].")
+			if(body_part != "chest")
+				to_chat(user, "<span class='warning'>Stethoscopes only work on the chest!</span>")
```

Comment:
You get just a pulse instead of the rushing air and blood sounds that you'd hear in the chest, I guess.